### ISSUE TITLE
Directory marked as excluded and added to ignore in VCS is hard to see.

### ISSUE THREAD

**Issue by lxzxl** [ChrisRM/material-theme-jetbrains#375]
When a directory is marked as excluded and is added to ignored in VCS, the files names under this directory is hard to see.
![webstorm-bug](https://user-images.githubusercontent.com/2275002/27851379-d564166a-618c-11e7-88a3-18aa7e7b8886.png)

**Comment by mallowigi** [comment]
You probably have the gitignore extension right? they have an option to color ignored files to gray, which is making this issue.

In any case you can change the color of the excluded files under Options > Appearance > File Colors. You have to move up the "Material XXX" file color to the top to take priority.

**Comment by lxzxl** [comment]
@mallowigi , thanks for your suggestion, but after I removed the ignore plugin and restarted, it still has the problem.

![image](https://user-images.githubusercontent.com/2275002/27866667-d04d8368-61c9-11e7-94f5-763f5606087f.png)

**Comment by mallowigi** [comment]
This is odd... maybe you have some file colors installed? Can you give me the list of plugins you use?

**Comment by lxzxl** [comment]
@mallowigi , I changed the first Material Default custom color which solved the problem in file colors.
![image](https://user-images.githubusercontent.com/2275002/27876986-9d3a4048-61eb-11e7-8285-b543c5556337.png)

**Comment by mallowigi** [comment]
Still it doesn't explain why the text color is gray... 

Wait are you still in 2016 versions of the IDE?

**Comment by lxzxl** [comment]
No I'm using the latest version. Anyway, the problem is missing after I totally remove the theme plugin and the file colors and re-installed. I guess it was introduced by the EAP plugin because there are more than 4 file colors in the list? I ever installed the EAP version plugin.

**Comment by mallowigi** [comment]
Yep, as stated in https://github.com/ChrisRM/material-theme-jetbrains#file-colors

To simplify, at the start of the application the plugin is looking for four themes in file colors with their default values. If it doesn't find one (either because you removed or changed one of them), it will create a new version with the default colors. Of course a better way would be to bundle them with the plugin in the same flavor as the color schemes do, but this is not an option offered by IDEA, so the only workaround would be what I did

As for the text color, I still don't understand why you have gray text. There should be some option for it, I will look for it.

TL;DR: Just use your own "Material Excluded File Color", set it at the top, and leave it as it is. Someday I will work on customizable file colors so you shouldn't have gray text.
